### PR TITLE
Add question counter

### PR DESCRIPTION
**Pull request by aleezaarc**
Based off of feature/undo branch

### CODE REVIEW

**Comment by SebastianKapunkt** [comment]
@aleezaarc this should not be based on the undo branch. Just rebase on master :)

**Comment by amureki** [comment]
@aleezaarc @SebastianKapunkt 
It would be nice to cover new context data covered in the `TestAnswerCreateView`. 😉

File: threesixty/views.py [hunk 1]
```diff
@@ -277,10 +277,16 @@ def get_question(self):
             raise models.Question.DoesNotExist('No question found.')
 
     def get_context_data(self, **kwargs):
+        qs = models.Question.objects
+        total_questions = qs.count()
+        answered_questions = answered = self.participant.answer_set.all().values_list('question_id', flat=True).count()
```
Comment (amureki) [inline threesixty/views.py]: Seems like `answered` is declared but is not used here.

So we can remove it, I guess, leaving only `answered_questions` declaration.
Also, `values_list` would not make a big difference here, so we can omit it.

File: threesixty/tests/test_views.py [hunk 2]
```diff
@@ -316,3 +326,29 @@ def test_form_valid_skip_allowed(self, client, db):
...
+        answer = self.create_answer(question1, survey, participant)
+        answer.save()
+
+        response = client.get(participant.get_absolute_url())
+
+        assert response.status_code == 200
+        assert response.context['name'] == survey.employee_name
+        assert response.context['statement'] == question2.text
+        assert response.context['answered_questions'] == 1
+        assert response.context['total_questions'] == 2
```
Comment (codingjoe) [inline threesixty/tests/test_views.py]: missing new line

File: threesixty/tests/test_views.py [hunk 3]
```diff
@@ -34,6 +34,14 @@ def create_question(self):
         )
         return question
 
+    def create_answer(self, question, survey, participant):
```
Comment (codingjoe) [inline threesixty/tests/test_views.py]: I would add `**kwargs` here to have a proper factory. sth like
```python
def create_answer(self, question, survey, participant, **kwargs):
    return Answer(
        survey=survey,
        question=question,
        participant=participant,
        **kwargs,
    )
```

**Comment by aleezaarc** [comment]
I am a bit lost with testing templates. Is there a good example of this somewhere? @codingjoe

**Comment by codingjoe** [comment]
go for it 👍

**Pull request merged**